frontend/public/locales/en/translation.json

```diff
@@ -1948,6 +1949,7 @@
     "Show only:": "Show only:",
     "show.less": "Show less",
     "show.more": "{{number}} more",
+    "Some policies have a Prune Object Behavior set": "Some policies have a Prune Object Behavior set",
     "Some resources failed to deploy. Use View resource YAML link below to view the details.": "Some resources failed to deploy. Use View resource YAML link below to view the details.",
```

Comment:
Some resources failed to deploy. Use the _View resource YAML_ link to view details. 

Remove the words like below, above, left, right, etc.